dockergen/nginx.tmpl

```diff
@@ -71,6 +72,7 @@ server {
 {{ range $url, $containers := groupByMulti $containersByHost "Env.VIRTUAL_URL" "," }}
     location {{ $url }} {
         proxy_pass http://{{ $host }}{{ replace $url "/" "-" -1 }}u/;
+        proxy_redirect https://{{ $host }}{{ replace $url "/" "-" -1 }}u/ /;
```

Comment:
Probably it should be `proxy_redirect http://{{ $host }}{{ replace $url "/" "-" -1 }}u/ {{ $url }};`.

Which would make it the same as `proxy_redirect default`, no? Which is default.